app/assets/javascripts/spree/backend/translations.js.coffee

```diff
@@ -30,7 +30,7 @@ $ ->
     display_locale_fields()
     false
 
-  $('#locale, #supported_locales_').select2({placeholder: Spree.translations['please_choose_language']})
+  $('#locale, #available_locales_').select2({placeholder: Spree.translations['please_choose_language']})
```

Comment:
These gems used to be one gem before. So, this feature may seem a little confusing (it was for me). `available_locales` (provided by `solidus_i18n`) defines the locales the store supports in a whole, while `supported_locales` (provided by `solidus_globalize`) defines the translatable languages available to admins. 

So `supported_locales` are always subset of the `available_locales`.

That being said. By changing this code you re-initialize the `select2` select box replacement on the select box provided by the `solidus_i18n` gem, which this gem depends on.

So, this code here needs to stay like it is.